3.7.pythonhome-excludes

```diff
@@ -0,0 +1,17 @@
+# This file is used by `zip -i@this_file` in `Dockerfile` on
+# each architecture to filter the Python standard library we
+# distribute.
+lib/python*/config-*
+lib/python*/ctypes/test/*
+lib/python*/curses/*
+lib/python*/distutils/tests/*
+lib/python*/ensurepip/*
+lib/python*/idlelib/*
+lib/python*/lib2to3/tests/*
+lib/python*/site-packages/*
+lib/python*/sqlite3/test/*
```

Comment:
I think we can leave sqlie3 in for Android by default?

Reply:
This only removes the sqlite3 test files, not the sqlite3 library itself. For that reason, I vote we keep this.

Reply:
🤦‍♂️

Of course. My mistake.